<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>WARNINGGG</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Server Information</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QListWidget" name="lstConsole"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Port</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Message Information</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Message</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lnMsg"/>
             </item>
             <item>
              <widget class="QPushButton" name="BtnSendToAll">
               <property name="text">
                <string>Send to All</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="Connection_Status">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#ConnectionStatus [state=&quot;0&quot;]{
background-color: rgb(255, 0, 0);
}
#ConnectionStatus [state=&quot;1&quot;]{
background-color: rgb(85, 255, 0);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="state" stdset="0">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QSpinBox" name="Spn_Server_Port">
            <property name="minimum">
             <number>1024</number>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QPushButton" name="Button_Start_Server">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 0);</string>
            </property>
            <property name="text">
             <string>Start Server</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="Btn_Clear">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Btn_Clear</sender>
   <signal>clicked()</signal>
   <receiver>lstConsole</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
